IBPSA/BoundaryConditions/BESTEST.mo

```diff
@@ -0,0 +1,1718 @@
+          color={255,204,51},
+          thickness=0.5));
+      connect(Azi090Til30.weaBus, Azi315Til90.weaBus) annotation (Line(
+          points={{-58,-46},{0,-46},{0,78},{-58,78}},
+          color={255,204,51},
+          thickness=0.5));
+      connect(weaBus.pAtm,x_pTphi. p_in) annotation (Line(
+          points={{-1,-68},{-32,-68},{-32,-78},{-44,-78}},
+          color={255,204,51},
+          thickness=0.5), Text(
+          string="%first",
+          index=-1,
+          extent={{-3,-6},{-3,-6}},
+          horizontalAlignment=TextAlignment.Right));
+      connect(weaBus.TDryBul,x_pTphi. T) annotation (Line(
+          points={{-1,-68},{-20,-68},{-20,-84},{-44,-84}},
+          color={255,204,51},
+          thickness=0.5), Text(
+          string="%first",
+          index=-1,
+          extent={{6,3},{6,3}},
+          horizontalAlignment=TextAlignment.Left));
+      connect(weaBus.relHum,x_pTphi. phi) annotation (Line(
+          points={{-1,-68},{-16,-68},{-16,-90},{-44,-90}},
+          color={255,204,51},
+          thickness=0.5), Text(
+          string="%first",
+          index=-1,
+          extent={{6,3},{6,3}},
+          horizontalAlignment=TextAlignment.Left));
+      connect(x_pTphi.X[1], toDryAir.XiTotalAir) annotation (Line(points={{-67,
+              -84},{-70,-84},{-70,-85},{-72.9,-85}}, color={0,0,127}));
+      connect(weaDat1.weaBus, weaBusDew) annotation (Line(
+          points={{48,-76},{48,-66},{47,-66}},
+          color={255,204,51},
+          thickness=0.5), Text(
+          string="%second",
+          index=1,
+          extent={{-3,6},{-3,6}},
+          horizontalAlignment=TextAlignment.Right));
+      annotation (
+      Documentation(info="<html>
+<h4>WD600: Ground Reflactance</h4>
+<p>Weather data file : 725650.epw</p>
+<p><i>Table 1: Site Data for Weather file 725650.epw</i></p>
+<table cellspacing=\"2\" cellpadding=\"0\" border=\"1\"><tr>
+<td><p>Latitude</p></td>
+<td><p>39.833&deg; north</p></td>
+</tr>
+<tr>
+<td><p>Longitude</p></td>
+<td><p>104.65&deg; west</p></td>
+</tr>
+<tr>
+<td><p>Altitude</p></td>
+<td><p>1650 m</p></td>
+</tr>
+<tr>
+<td><p>Time Zone</p></td>
+<td><p>7</p></td>
+</tr>
+</table>
+<p><br><h4>Additional parameters and correlations:</h4></p>
+<p>Ground reflectance &rho; is set to 0.2</p>
+<p>Black body temperature calculated using Horizontal radiation or dew point temperature and sky cover</p>
+<p>Perez or Isoentropic sky model used</p>
+</html>",     revisions="<html>
+<ul>
+<li>
+March 11, 2020, by Ettore Zanetti:<br/>
+First implementation.
+</li>
+</ul>
+</html>"),
+    experiment(
+          StopTime=31539600,
+          __Dymola_NumberOfIntervals=35043,
+          Tolerance=1e-06,
+          __Dymola_Algorithm="Dassl"),
+    __Dymola_Commands(file="modelica://IBPSA/Resources/Scripts/Dymola/BoundaryConditions/SolarGeometry/Examples/IncidenceAngle.mos"
+            "Simulate and plot"));
+    end WD600;
+
+  end Validation;
+
+  model ComRadIsoPerDir
```

Comment:
Please use a full name (camel case is used for instances, not for class names).

Reply:
Added full name "IsotropicAndPerezDiffuseRadiation" instead of camelcase